src/process/status.cr

```diff
@@ -181,13 +182,12 @@ class Process::Status
   end
 
   # Returns `true` if the process terminated normally.
+  #
+  # Equivalent to `ExitReason::Normal`
+  #
+  # * `#exit_reason` provides more insights into other exit reasons.
```

Comment:
Tiny suggestion:

```suggestion
  # NOTE: `#exit_reason` provides more insights into other exit reasons.
```

Reply:
I don't think an admonition improves anything here. We're using the bullet point syntax for "See also" references in other doc comments and this follows that established format (eventually, we might want a more explicit, dedicated syntax for that).

Reply:
_See also_ references were usually written verbatim AFAIR: see these [search results](https://github.com/search?q=repo%3Acrystal-lang%2Fcrystal%20%22see%20also%22&type=code).

Reply:
There are instances of different variants, all of them work (but none is really great) and we haven't standardized on anything. So let's not bother about that in this PR.

Reply:
Still, it's gonna be way easier to search for "See also" phrase in the future, once the syntax is gonna be introduced. Such bullet points might be easily missed among false positives.